script.module.inputstreamhelper/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Please remove this file from the PR

Reply:
Okay, I removed this and UTF-8 was fixed after force pushing above.